include/options.hxx

```diff
@@ -281,6 +283,32 @@ public:
   const Options& operator[](const std::string &name) const;
   const Options& operator[](const char *name) const { return (*this)[std::string(name)]; }
 
+  /// Return type of `Options::fuzzyFind`
+  struct FuzzyMatch {
+    /// Full name (including parent sections) of possible match
+    std::string name;
```

Comment:
This might actually be nicer as the `Options` itself (or a copy), rather than just the name